codebase/codebase.go

```diff
@@ -140,7 +140,7 @@ type Repository interface {
 	//Save(ctx context.Context, codebase *Codebase) (*Codebase, error)
 	List(ctx context.Context, spaceID uuid.UUID, start *int, limit *int) ([]*Codebase, uint64, error)
 	Load(ctx context.Context, id uuid.UUID) (*Codebase, error)
-	LoadByRepo(ctx context.Context, repository string) (*Codebase, error)
+	LoadByRepo(ctx context.Context, repository string, spaceID uuid.UUID) (*Codebase, error)
```

Comment:
Thank @aslakknutsen  but yy thought was to align it with `Load` since this is a `LoadByRepo` method.
Second arguments tells on what column search will takes place.
But, modified to align with `List` in https://github.com/almighty/almighty-core/pull/1241/commits/60e6318c8d368d5a6b3caff8d2613093461003e2